sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyALM.java

```diff
@@ -1969,4 +1969,22 @@ public boolean isAppResumeSuccess() throws SdlException {
 		return _bResumeSuccess;
 	}
 
+	/**
+	 * Tells developer whether or not their app icon has been resumed on core.
+	 * @return boolean - true if icon was resumed, false if not
+	 * @throws SdlException if proxy is disposed or app is not registered
+	 */
+	public boolean getIconResumed() throws SdlException {

```

Comment:
I believe this should be part of SdlProxyBase. This class has essentially been all moved into the base class.